config/autoload/local.php.dist

```diff
@@ -1,11 +1,12 @@
 <?php
-
 /**
  * Local configuration.
  *
  * Copy this file to `local.php` and change its settings as required.
  * `local.php` is ignored by git and safe to use for local and sensitive data like usernames and passwords.
  */
 
+declare(strict_types=1);
```

Comment:
I'm not sure where it should be, maybe before the comment?

In PHPCode_Sniffer I've added functionality to put it after the comment, but only if comment has specific tags.
Here there is no any tags in the comment, so by my sniff declaration will be placed before it.

Reply:
According to the example in [PSR-12](https://github.com/php-fig/fig-standards/blob/master/proposed/extended-coding-style-guide.md#3-declare-statements-namespace-and-import-statements) it should be after the file docblock.

Reply:
It could be hard to detect if the doc block is file or class level in some cases.... but fine then, maybe I need improve my sniff.